### PR TITLE
xLaronx Locations

### PR DESCRIPTION
**Pull request by xLaronx**

### CODE REVIEW

File: Translation/en/Text/Override.txt [hunk 1]
```diff
@@ -921,6 +921,28 @@ sr:"数术术法精进提升([\+\=\d\.\%]*)"=Management Skill Improvement $1
...
-
-
-
+#Random Stuff(Laron):
+
+藏经阁=Scripture Pavillon
+百草园=Herb Field
+读档=Load  
+存档=Save
+<color\=red>身陨道消</color>=<color\=red>High Risk of Death(?)</color>
```
Comment (pipja) [inline Translation/en/Text/Override.txt:1093]: Call it `Dangerous` or something?

File: Translation/en/Text/Override.txt [hunk 2]
```diff
@@ -921,6 +921,28 @@ sr:"数术术法精进提升([\+\=\d\.\%]*)"=Management Skill Improvement $1
...
+大风国=Great Wind Country
+古商国=Ancient Merchant Country
+荒神堂=Temple of the famine god
```
Comment (pipja) [inline Translation/en/Text/Override.txt]: `Temple of the Famine God` maybe do a bit of proper capitalisation?

File: Translation/en/Text/Override.txt [hunk 3]
```diff
@@ -921,6 +921,28 @@ sr:"数术术法精进提升([\+\=\d\.\%]*)"=Management Skill Improvement $1
...
+生死派=Life and death Faction(?)
```
Comment (pipja) [inline Translation/en/Text/Override.txt]: Same here

File: Translation/en/Text/Override.txt [hunk 4]
```diff
@@ -921,6 +921,28 @@ sr:"数术术法精进提升([\+\=\d\.\%]*)"=Management Skill Improvement $1
...
+花啸派=Rustling Flower(s) Sect
+雨旁国=Rain Country
+无情门=Ruthless Gate
+宵顺城=Smooth Night City?
+盘苍帮=Pancang Gang (ancient Chinese people's gang)
```
Comment (pipja) [inline Translation/en/Text/Override.txt:1102]: This is something along the lines of a `Transport & Trade Co.`

File: Translation/en/Text/Override.txt [hunk 5]
```diff
@@ -921,6 +921,28 @@ sr:"数术术法精进提升([\+\=\d\.\%]*)"=Management Skill Improvement $1
...
+赤霄教=Red Sky Church
+碧劫派=Bijou Faction
```
Comment (pipja) [inline Translation/en/Text/Override.txt]: `Red Destiny Faction`

**Review by pipja — APPROVED** [review]
Some suggestions, you may merge it or make some updates then merge it.

Cheers

File: Translation/en/Text/Override.txt (hunk 1 above)
Comment (xLaronx) [reply Translation/en/Text/Override.txt:1093]: Or Risky its only for auto Battle

Comment (xLaronx) [reply Translation/en/Text/Override.txt:1093]: High Risk

**Review by pipja — APPROVED** [review]

**Pull request merged**